Documentation/data-collection.md

```diff
@@ -145,7 +145,14 @@ data:
     # version, phase, reason labels. When a csv is updated, the previous time
     # series associated with the csv will be deleted.
     - '{__name__="csv_abnormal"}'
-    # OCS metrics to be collected:
+    # Storage metrics:
+    # cluster:storage_provisioner_capacity_requested:total will give the total amount of
```

Comment:
do you mind to add team owners for the new team metrics? there seems to be just one entry prefexing the ceph ones.

Reply:
Sure.